projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-abstract-token-subscription-service.php

```diff
@@ -472,7 +472,7 @@ public static function clear_token_cookie() {
 	private function token_from_request() {
 		$token = null;
 		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-		if ( isset( $_GET['token'] ) ) {
+		if ( isset( $_GET['token'] ) && isset( $_GET['token'] ) ) {
```

Comment:
```suggestion
		if ( isset( $_GET['token'] ) && is_string( $_GET['token'] ) ) {
```

I think we might have a small typo here.

Reply:
You are correct, thank you for noticing that.